src/utils/CMakeLists.txt

```diff
@@ -1,11 +1,20 @@
 # build version string from git query
 execute_process(COMMAND git rev-list --max-count=1 HEAD
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/..
                 OUTPUT_VARIABLE PTEX_SHA
                 OUTPUT_STRIP_TRAILING_WHITESPACE)
 execute_process(COMMAND git describe ${PTEX_SHA}
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/..
```

Comment:
alignment seems off. I think `${PROJECT_SOURCE_DIR}` is probably better for the working directory.  Git can do its discovery from there.